### ISSUE TITLE
NTR: gamma-glutamylcysteine high in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8503]
New term request gamma-glutamylcysteine high in urine (segal_030502164138):

**Comment by pnrobinson** [comment]
@MickeySegal
I am not sure this is a clinical assay. I tried to search for normal range or medical applications but could not find anything. Could you provide context?

**Comment by MickeySegal** [comment]
https://ojrd.biomedcentral.com/articles/10.1186/1750-1172-2-16 "patients have increased urinary levels of gamma-glutamylcysteine and cysteine"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000834
- label: Elevated urinary gamma-glutamylcysteine level
- parents: [Increased urinary non-proteinogenic amino acid level (HP:0033098)](https://hpo.jax.org/app/browse/term/HP:0033098)
- pmid: PMID:17397529
- def: The amount of gamma-glutamylcysteine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8503
- synonyms:
